docs/code/MlNetCookBook.md

```diff
@@ -508,24 +512,22 @@ var trainData = mlContext.Data.LoadFromEnumerable(churnData);
 // We apply our FastTree binary classifier to predict the 'HasChurned' label.
 
 var pipeline =
-    // Convert each categorical feature into one-hot encoding independently.
-    mlContext.Transforms.Categorical.OneHotEncoding("CategoricalOneHot", "CategoricalFeatures")
+    // Convert the categorical feature 'DemographicCategory' into one-hot encoding.
+    mlContext.Transforms.Categorical.OneHotEncoding("DemographicCategoryOneHot", "DemographicCategory")
     // Convert all categorical features into indices, and build a 'word bag' of these.
-    .Append(mlContext.Transforms.Categorical.OneHotEncoding("CategoricalBag", "CategoricalFeatures", OneHotEncodingEstimator.OutputKind.Bag))
-    // One-hot encode the workclass column, then drop all the categories that have fewer than 10 instances in the train set.
-    .Append(mlContext.Transforms.Categorical.OneHotEncoding("WorkclassOneHot", "Workclass"))
-    .Append(mlContext.Transforms.FeatureSelection.SelectFeaturesBasedOnCount("WorkclassOneHotTrimmed", "WorkclassOneHot", count: 10));
+    .Append(mlContext.Transforms.Concatenate("Features", new[] { "DemographicCategoryOneHot", "LastVisits" })
+    .Append(mlContext.BinaryClassification.Trainers.FastTree());
 
 var model = pipeline.Fit(trainData);
 ```
 
 ## I want to look at my model's coefficients
 
-Oftentimes, once a model is trained, we are also interested on 'what it has learned'. 
+Oftentimes, once a model is trained, we are also interested in 'what it has learned'. 
 
 For example, if the linear model assigned zero weight to a feature that we consider important, it could indicate some problem with modeling. The weights of the linear model can also be used as a poor man's estimation of 'feature importance'.
 
-We provide a set of `onFit` delegates that allow introspection of the individual transformers as they are trained.
+We provide a set of `onFit` delegates that allow inspection of the individual transformers as they are trained.
```

Comment:
>onFit [](start = 21, length = 5)

This is mentioned here, but I don't see anywhere in this file that shows how to use it.

Reply:
This isn't really relevant to model coefficients. I will add another section for this.

---
In reply to: [370991105](https://github.com/dotnet/machinelearning/pull/4706#discussion_r370991105) [](ancestors = 370991105)